packages/plugin-ext/src/plugin/languages/diagnostics.ts

```diff
@@ -223,7 +223,8 @@ export class DiagnosticCollection implements theia.DiagnosticCollection {
                                         startLineNumber: lastMarker.startLineNumber,
                                         startColumn: lastMarker.startColumn,
                                         endLineNumber: lastMarker.endLineNumber,
-                                        endColumn: lastMarker.endColumn
+                                        endColumn: lastMarker.endColumn,
+                                        tags: lastMarker.tags,
```

Comment:
This is an artificial marker. I don't think we should don't think we should apply the tags for the marker it replaces.

Reply:
@tsmaeder thank you for clarifying it's use-case!